app/grants/clr.py

```diff
@@ -545,6 +550,8 @@ def predict_clr(save_to_db=False, from_date=None, clr_round=None, network='mainn
 
         debug_output.append({'grant': grant.id, "title": grant.title, "clr_prediction_curve": (potential_donations, potential_clr), "grants_clr": grants_clr})
 
+    invalid_clr_calculations.update(latest=False)
```

Comment:
We should also add `active=False` to this update 👍

Reply:
```suggestion
    invalid_clr_calculations.update(latest=False, active=False)
```

Reply:
> We should also add `active=False` to this update 👍